modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/modified/facet/portlet/ModifiedFacetPortlet.java

```diff
@@ -94,6 +94,11 @@ public void render(
 		renderRequest.setAttribute(
 			WebKeys.PORTLET_DISPLAY_CONTEXT, modifiedFacetDisplayContext);
 
+		if (modifiedFacetDisplayContext.isRenderNothing()) {
+			renderRequest.setAttribute(
+				WebKeys.PORTLET_CONFIGURATOR_VISIBILITY, Boolean.TRUE);
+		}
```

Comment:
Let's move this before the `renderRequest.setAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT...` to alphabetize the attributes.